test/index.js

```diff
@@ -154,13 +167,13 @@ test('fm(string) - no front matter, markdown with hr', function (t) {
     })
 })
 
-test('fm(string) - complex yaml', function (t) {
+test('fm(string, true) - complex yaml', function (t) {
```

Comment:
Nit: I would change part of the description here to be clearer, change "complex yaml" to "complex and unsafe yaml"